pennylane_lightning/core/src/simulators/lightning_kokkos/StateVectorKokkos.hpp

```diff
@@ -326,7 +330,7 @@ class StateVectorKokkos final
      * @param wires Wires to apply gate to.
      * @param inverse Indicates whether to use adjoint of gate.
      */
-    void applyMultiQubitOp(const KokkosVector &matrix,
+    void applyMultiQubitOp(const KokkosVector matrix,
```

Comment:
Why are we changing the way we pass the variable here?

Reply:
Kokkos views are trivially copyable, so I think it is simpler to pass them around by value.